src/style.css

```diff
@@ -1 +1,77 @@
 /* Styles go here */
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 14px;
+  box-sizing: border-box;
+}
+
+.header {
+  width: 1200px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  margin: 0;
+}
+
+.img__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.list__item {
+  text-transform: uppercase;
+  color: #000;
+  margin-right: 20px;
+}
+
+.list__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: #000;
+  cursor: pointer;
+}
+
+.nav__link:hover {
+  color: hsl(193, 100%, 43%);
+}
+
+.is-active {
+  color: hsl(193, 100%, 43%);
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: hsl(193, 100%, 43%);
+  border-radius: 8px;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
```

Comment:
Max width would be better here, ui would stop being cut off on small screens (see image)
```suggestion
    max-width: 1024px;
```

![image](https://github.com/mate-academy/layout_moyo-header/assets/23013525/2b1da03f-29f0-46ed-aec1-5ee6c4bae46d)